alacritty/src/event.rs

```diff
@@ -384,6 +386,23 @@ impl<N: Notify + OnResize> Processor<N> {
         }
     }
 
+    /// Dispatch wayland event queue
```

Comment:
```suggestion
    /// Dispatch wayland event queue.
```

Reply:
I'm not sure about this dot, we have a lot of code without dot in a top level comment.

Reply:
I'm sure about it. The 'headline' should always have a dot. You can see that pretty clearly when looking at Rust's official documentation for modules:

https://doc.rust-lang.org/std/index.html#primitives

Every other place is just doing it wrong. Though I do understand how this can be easily done wrong since headlines aren't usually punctuated with a trailing dot.